src/custom-types.json

```diff
@@ -50,6 +50,12 @@
       "http://www.iana.org/assignments/media-types/application/geo+json"
     ]
   },
+  "application/grpc": {
+    "extensions": ["grpc"],
```

Comment:
There is no file extension listed in the source provided. Please provide a source that states the file extension for this type is defined to be .grpc

Reply:
sorry, my mistake. should add `proto` as extension for `application/grpc+proto`